WL4EditorWindow.cpp

```diff
@@ -4,6 +4,7 @@
 #include "ROMUtils.h"
 #include "ui_WL4EditorWindow.h"
 #include "Themes.h"
+#include "LevelComponents/Room.h"
```

Comment:
this line not needed.

Reply:
It is needed for the __CameraControlType enum used on line 1955

Reply:
Okay it is not needed